### DISCUSSION TITLE
同一个pag资源 使用PAGImageView出现锯齿 PAGView不会 为什么？

### DISCUSSION THREAD

**Discussion by ph20000831** [Tencent/libpag#1722] [Q&A]
https://i.postimg.cc/0Q7CGJKN/2023-09-11-11-15-41.png
图片⬆️
左边为PAGView 右边为PAGImageView单独使用也会出现锯齿 只是放在一起对比想请问是资源问题还是使用方法问题

**Comment by liguobing** [comment]
同样的问题，哥们儿你有答案了吗？
